source/includes/_banks.md

```diff
@@ -282,6 +282,11 @@ Creating a business profile is similar to how you created personal profile. Ther
 
 2) [Create business user profile – address data](#addresses-create) 
 
+3) [Create business user profile - directors data](#user-profiles-add-business-directors)
+
+3) [Create business user profile - owners data](#user-profiles-add-business-ultimate-beneficial-owners)
+
+
 Currently we do not support a fully automated business onboarding flow over the API - this feature is coming soon, please contact the TransferWise team for more details.
```

Comment:
remove this message?

Reply:
The user needs to provide all the information about the profile, including directors and shareholders before we can make a transfer. Am I incorrect and we don't need shareholders?

Reply:
Done